tests/server-snap/task.yaml

```diff
@@ -0,0 +1,26 @@
+summary: Check snap web servers
+environment:
+  SNAP_NAME/pythonServer: xkcd-webserver
```

Comment:
Very nice use of the environment feature! We should definitely highlight some of these tests as good examples how to write nice and DRY tests :)